Blizzard.IO.RabbitMQ.Tests/Rpc/NetqRabbitRpcServerTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class RequestStub
+    {
+        public int Value { get; set; }
+    }
+
+    class RespondStub
+    {
+        public int Value { get; set; }
+    }
+
+    public class NetqRabbitRpcServerTests
+    {
+        private NetqRabbitRpcServer _server;
+        private Mock<INetqRpcRabbitConnection<Func<Type, object>>> _netqRpcRabbitConnectionMock;
+        private Mock<IBus> _busMock;
```

Comment:
```suggestion
        private Mock<INetqRpcRabbitConnection<Func<Type, object>>> _netqRpcRabbitConnectionMock;
        private Mock<IBus> _busMock;
        
        private NetqRabbitRpcServer _server;
```

It's more readable like this